src/assets/css/librarybrowser.scss

```diff
@@ -738,18 +733,34 @@
     }
 }
 
+[dir=ltr] .mainDetailButtons {
+    @media all and (max-width: 62.5em) {
+        margin-left: -0.5em;
+    }
+}
+
+[dir=rtl] .mainDetailButtons {
+    @media all and (max-width: 62.5em) {
+        margin-right: -0.5em;
+    }
+}
+
 .layout-mobile .parentName,
 .layout-mobile .itemName,
 .layout-mobile .itemMiscInfo,
 .layout-mobile .mainDetailButtons {
     align-items: center;
     justify-content: center;
-    text-align: center;
+    text-align: center !important;
```

Comment:
To avoid `important`, you can remove:
https://github.com/jellyfin/jellyfin-web/blob/ac105f4c831f3e9e30ea318d06f69af42a18a6bf/src/assets/css/librarybrowser.scss#L862-L869
`text-align` will be `start` (or just unset), which seems to make it aligned to the correct side.

Reply:
This doesn't work. Text should be centered, else it might be under the thumbnail.

Reply:
Did it. LMK if that does what you expected.